genie_annotations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tissueSource": [
+    "Blood",
+    "Solid Tumor"
+  ],
+  "center": [
+    "NKI",
+    "DFCI",
+    "GRCC",
+    "JHU",
+    "MSK",
+    "UHN",
+    "VICC",
+    "MDA"
+  ],
+  "platform": [
```

Comment:
I believe we discussed platform being controlled in the minified standard? If not we should.